store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/AddUpdateTableIterator.java

```diff
@@ -192,6 +198,16 @@ public static void main(final String[] args) throws Exception {
 
         final Schema schema = Schema.fromJson(getSchemaPaths(args));
 
+        GraphLibrary library;
+
+        if (getFileGraphLibraryPathString(args) == null) {
+            library = new NoGraphLibrary();
+        } else {
+            library = new FileGraphLibrary(getFileGraphLibraryPathString(args));
+        }
+
+        library.add(getGraphId(args), schema, storeProps);
```

Comment:
We should call addOrUpdate as we want to update the schema and store properties. Similarly for HBase.